### ISSUE TITLE
Consolidated list of potential new mods for 2.7.0

### ISSUE THREAD

**Issue by mitchej123** [GTNewHorizons/GT-New-Horizons-Modpack#16436]
### Your GTNH Discord Username

mitchej123

### Your Pack Version

2.6.1

### Mod Name

Multiple

### The Mod to Add

Consolidating several tickets here:
- [ ] [Et-Future Requiem](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/13627)
- [ ] [Village Names](https://www.curseforge.com/minecraft/mc-mods/village-names)
- [ ] [Baubles Expanded](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/14378)
- [ ] [Alfheim](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/9686)
- [ ] [Modern Controlling, ModernKeyBinding, KeyboardWizard-Legacy](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/16163)

### Your Reason

EFR & Village Name backport multiple things from future versions, mostly blocks and QoL additions.  They're configurable as to which features are enabled, and should be backwards compatible with existing worlds.  EFR adds ocean monuments, but no guardians -- VN adds both.

Baubles Expanded adds more baubles, is configurable, FOSS, and would be a good ultimate replacement for Traveler's Gear (with some additions there or in other mods)

Alfheim - Someone else suggested it, seems like a cool idea

### Preliminary Investigation

I've gotten all of the mods running locally on a new playthrough.

Current notes:
* EFR Required config change: `etfuturm/enchantspotions.cfg` - Fix Enchant ID conflicts
```
enchantments {
    #  [default: true]
    B:frostWalker=true

    #  [range: 0 ~ 32767, default: 200]
    I:frostWalkerID=201

    #  [default: true]
    B:mending=true

    #  [range: 0 ~ 32767, default: 201]
    I:mendingID=202

    #  [default: true]
    B:swiftSneak=true

    #  [range: 0 ~ 32767, default: 202]
    I:swiftSneakID=203
}
```
* EFR Required config change: `etfuturum/blocksitems.cfg`  - Disable Prismarine (Use Botania's)  Causes a crash with the null recipe checker for inpure dust and others if not disabled
```
"natural blocks" {
     #  [default: true]
     B:enablePrismarine=false
}
```
* VN Required config change: `VillageNames4/general.cfg` - Disable Prismarine (same reason as EFR)
```
miscellaneous {
    # Adds Prismarine blocks and items. Very difficult to acquire if "Add Monuments" is false. [default: true]
    B:"Add Prismarine"=false

}
```
* Alfheim Required config change: `Alfheim/Alfheim.cfg` - Disable MMO Mode
```
general {
   preload {
       # Set this to false to disable ESM and MMO
      B:enableElvenStory=false

       # Set this to false to disable MMO
       B:enableMMO=false
   }
}
```
* ModernKeybinding/etc - Disable HodgePodge mixin to trigger all conflicting keybinds - they have this functionality (Should update hodgepodge to auto disable this if they're present)
```
B:triggerAllConflictingKeybindings=false
```

### Final Checklist

- [X] This mod is released under an open source license. Yes, this is required now.
- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying *I like this mod, please reconsider adding it* will prompt us to investigate and reopen it once we confirm your report.
- [X] I believe there is nothing similar in the pack already, or the existing solution isn't good enough.
- [X] I understand this change request may not attract enough attention and thus not be implemented.
- [X] I understand this change request may be rejected due to other community members thinking it's inappropriate.
- [X] I believe this feature would make the pack better.
- [ ] I'm the official maintainer or original author of this mod for minecraft 1.7.10.

**Comment by chochem** [comment]
EFR or Alfheim would need a clear plan on how we want to integrate them and someone that wants to do that.

**Comment by mitchej123** [comment]
Alfheim sure 

EFR - Most of the blocks added are early game/vanilla blocks mostly added via worldgen, is there anything specific you're thinking of?

**Comment by chochem** [comment]
well I still dont know whats actually in EFR. maybe post the config somewhere if thats the only overview.
but I guess netherite and elytra would be two examples. like maybe removing some other clunkier flight options and gregifying rockets appropriately and who know what we want to do with netherite.

**Comment by mitchej123** [comment]
```
# Configuration file

##########################################################################################################
# entity items
#--------------------------------------------------------------------------------------------------------#
# Entity items. (Armor stand, boat, etc)
##########################################################################################################

"entity items" {
    #  [default: true]
    B:enableArmorStand=true

    # New boats from 1.9+, including the new rowing sounds. All vanilla wood variants included. [default: true]
    B:enableNewBoats=true

    # What entities shouldn't be able to sit in the boat? You can either provide an entity ID (modid.entityid, for vanilla entities type just entity ID), or search for a string in the classpath (classpath:stringtofind).
    # Separate entries in the list by a new line. Note that players can always sit even if blacklisted, and some entities, like horses, water mobs or nonliving entities, will never be allowed to sit in boats.
    # It's a little hard to explain, a more detailed explanation and list of examples can be found here: https://pastebin.com/XNZ7VWKh
    S:newBoatEntityBlacklist <
     >

    # Treat the entity blacklist as a whitelist, ONLY entities matching that criteria will be allowed. [default: false]
    B:newBoatEntityBlacklistAsWhitelist=false

    # The maximum speed a boat can travel by while on land. This option exists because boats are very very fast when travelling on slippery blocks. Land speed = 0.6, Regular/Packed Ice Speed = 0.98, Packed Ice Speed = 0.986. Anything smaller than 0.6 is really, REALLY slow on land.
    # The speed values are just block slipperiness values, and are averaged by the slippery blocks around the bottom of the boat. This option does nothing to old boats. [range: 0.1 ~ 1.0, default: 0.986]
    S:newBoatMaxLandSpeed=0.986

    # If disabled, only one person can sit in the new boat at a time. The new seat is actually an invisible entity that follows new boats. [default: true]
    B:newBoatPassengerSeat=true

    # The speed multiplier for boats while in water. Use this if you want to make the boats faster or slower. 1 = no speed change [range: 0.1 ~ 2.0, default: 1.0]
    S:newBoatSpeed=1.0

    # If true, old boats will be replaced with the new oak boat and the item sprite will also be changned. False means the new and old boat and item for it exists separately, and the new boats will use a wooden shovel in their crafting recipe. If this is enabled, a boat that has an entity in it will not be replaced until the entity gets out.
    # THIS WILL NOT WORK PROPERLY WITH BETTER BOATS INSTALLED [default: true]
    B:replaceOldBoats=true
}


##########################################################################################################
# equipment
#--------------------------------------------------------------------------------------------------------#
# Tools, armor, and other equipment items.
##########################################################################################################

equipment {
    #  [range: 1 ~ 2147483647, default: 37]
    I:netheriteArmourDurabilityFactor=37

    # Override Netherite Axe Durability, -1 to disable [range: -1 ~ 2147483647, default: -1]
    I:netheriteAxeDurability=-1

    # Override Netherite Boots Durability, -1 for no override [range: -1 ~ 2147483647, default: -1]
    I:netheriteBootsDurability=-1

    # Netherite Boots Protection [range: 1 ~ 2147483647, default: 3]
    I:netheriteBootsProtection=3

    # Override Netherite Chestplate Durability, -1 for no override [range: -1 ~ 2147483647, default: -1]
    I:netheriteChestplateDurability=-1

    # Netherite Chestplate Protection [range: 1 ~ 2147483647, default: 8]
    I:netheriteChestplateProtection=8

    # Neterite base damage, Diamond is 3.0 [range: 0.0 ~ 3.4028235E38, default: 4.0]
    S:netheriteDamageBase=4.0

    #  [range: 1 ~ 2147483647, default: 15]
    I:netheriteEnchantability=15

    # Netherite harvest level, Diamond is 3 [range: 0 ~ 2147483647, default: 4]
    I:netheriteHarvestLevel=4

    # Override Netherite Helmet Durability, -1 to disable [range: -1 ~ 2147483647, default: -1]
    I:netheriteHelmetDurability=-1

    # Netherite Helmet Protection [range: 1 ~ 2147483647, default: 3]
    I:netheriteHelmetProtection=3

    # Override Netherite Hoe Durability, -1 to disable [range: -1 ~ 2147483647, default: -1]
    I:netheriteHoeDurability=-1

    # Override Netherite Leggings Durability, -1 for no override [range: -1 ~ 2147483647, default: -1]
    I:netheriteLeggingsDurability=-1

    # Netherite Leggings Protection [range: 1 ~ 2147483647, default: 6]
    I:netheriteLeggingsProtection=6

    # Override Netherite Pickaxe Durability, -1 to disable [range: -1 ~ 2147483647, default: -1]
    I:netheritePickaxeDurability=-1

    # Override Netherite Shovel Durability, -1 to disable [range: -1 ~ 2147483647, default: -1]
    I:netheriteSpadeDurability=-1

    # Netherite mining speed, Diamond is 8.0 [range: 0.1 ~ 3.4028235E38, default: 9.0]
    S:netheriteSpeed=9.0

    # Override Netherite Sword Durability, -1 to disable [range: -1 ~ 2147483647, default: -1]
    I:netheriteSwordDurability=-1

    #  [range: 1 ~ 2147483647, default: 2031]
    I:netheriteToolDurability=2031
}


##########################################################################################################
# function blocks
#--------------------------------------------------------------------------------------------------------#
# Blocks that have a specific function, whether right clicked or otherwise.
##########################################################################################################

"function blocks" {
    #  [default: true]
    B:enableBarrel=true

    # Will attempt to seek and auto-add recipes to itself. Look at ConfigFunctions.cfg "autoAddBlastFurance" for more info.
    # Compatible with CraftTweaker. In the same way that you'd use furnace.addRecipe or furnace.remove, you can use "mods.etfuturum.blastFurnace" instead of "furnace". [default: true]
    B:enableBlastFurnace=true

    # Currently DOES NOT HAVE ANY FUNCTIONALITY. Decoration ONLY! [default: true]
    B:enableCartographyTable=true

    # Allows certain items to be composted, which has a chance of adding a layer to the compost bin. Once full, bone meal may be harvested. It is compatible with CraftTweaker.
    # A value greater than 100 can add more than one layer to the composter. (The composter has 6 layers so max value is 600) EG 150 = 1 layer guaranteed and a 50% chance to fill another layer. And 600 = instantly fill the whole composter, or any remaining layers.
    # The mod prefix is "mods.etfuturum.composting", and the function is "addCompostable" and an ItemStack or OreDictionary tag, then an integer for how likely the item is to add a compost layer. 100 = 100%, or guaranteed. You can remove compostables using the "remove" function and an ItemStack or an OreDictionary tag.
    # You can use the "removeAll" function to remove all composting recipes, if you wanted to overhaul the composter's useage.
    # Examples:
    # mods.etfuturum.composting.addCompostable(<minecraft:planks:*>, 100); //(Makes all planks have a 100% chance to fill one layer.)
    # mods.etfuturum.composting.addCompostable(<minecraft:bedrock>, 150); //(Makes bedrock fill one layer and have a 50% chance to fill another layer.)
    # mods.etfuturum.composting.remove(<minecraft:leaves:*>); //(Removes all leaves as a compostable item)
    # 
    # Additionally the bone meal drop when harvesting a full composting bin can be changed. It drops one item from the "composting" loot table, and can be modified with CraftTweaker's "addChestLoot" function.
    # By default bone meal is the only item in this loot table, with a weight of 10. You can learn more about CraftTweaker loot table modification here: https://www.mcdrama.net/archiveformal/MineTweaker3/wiki/tutorials/1710/loot_and_seeds.html [default: true]
    B:enableComposter=true

    # Enables wood variant doors [default: true]
    B:enableDoors=true

    # Ability to craft differently colored beds out of wool. Mixed wool colors = red bed [default: true]
    B:enableDyedBeds=true

    # Currently DOES NOT HAVE ANY FUNCTIONALITY. Decoration ONLY! [default: true]
    B:enableFletchingTable=true

    # Inverted Daylight Sensor [default: true]
    B:enableInvertedSensor=true

    #  [default: true]
    B:enableIronTrapdoor=true

    # Allow lava buckets to fill cauldrons [default: true]
    B:enableLavaCauldrons=true

    # Currently DOES NOT HAVE ANY FUNCTIONALITY. Decoration ONLY! [default: true]
    B:enableLoom=true

    # Enables new anvil behavior, such as less expensive item renaming [default: true]
    B:enableNewAnvil=true

    # Beacon beam can be colored using stained glass [default: true]
    B:enableNewBeacon=true

    # Makes the brewing stand have a fuel slot like in 1.9+. The fuel slot is compatible with CraftTweaker and takes blaze powder by default. Blaze powder can brew 30 potion cycles.
    # The mod prefix is "mods.etfuturum.brewingFuel", and the function is "addFuel" and an ItemStack or OreDictionary tag, then an integer for how many brew cycles. (Any brewing, regardless of if 1 or all 3 slots are filled, is still one "cycle") You can remove fuels using the "remove" function and an ItemStack or an OreDictionary tag.
    # Examples:
    # mods.etfuturum.brewingFuel.addFuel(<minecraft:gunpowder>, 10); //(Makes gunpowder have 10 brewing cycles)
    # mods.etfuturum.brewingFuel.remove(<minecraft:blaze_powder>); //(Removes blaze powder as a brewing fuel) [default: true]
    B:enableNewBrewingStand=true

    # Uses lapis as payment and has enchant previews and adjusted level costs. Requires tile entity replacement to be enabled in "function.cfg". It is compatible with CraftTweaker for adding and removing fuels.
    # The mod prefix is "mods.etfuturum.enchantingFuel", and the function is "addFuel" and an ItemStack or OreDictionary tag. You can remove fuels using the "remove" function and an ItemStack or an OreDictionary tag.
    # Examples:
    # mods.etfuturum.enchantingFuel.addFuel(<etfuturum:amethyst_shard>); //(Adds amethyst shards as an enchanting fuel)
    # mods.etfuturum.enchantingFuel.remove(<minecraft:dye:4>); //(Removes lapis lazuli as an enchanting fuel) [default: true]
    B:enableNewEnchantingTable=true

    # Enable the old Et Futurum daylight sensor block. Should be enabled if you still have the old Et Futurum copy of the non-inverted daylight detector that need to be converted. [default: false]
    B:enableOldBaseDaylightSensor=false

    # A port of potion cauldrons from Bedrock Edition. Used to make tipped arrows and store potions. [default: true]
    B:enablePotionCauldron=true

    # If Shulkers are disabled, a custom recipe will be required to obtain Shulker shells. [default: true]
    B:enableShulkerBoxes=true

    #  [default: true]
    B:enableSigns=true

    # Just bouncy, does not pull blocks. [default: true]
    B:enableSlimeBlock=true

    # If this is disabled, netherite items will not be craftable unless added by CraftTweaker. This introduces the smithing GUI from versions prior to 1.20. It is compatible with CraftTweaker.
    # The mod prefix is "mods.etfuturum.smithingTable", and the functions are "addRecipe" or "addRecipeNoNBT". "addRecipe" will copy all NBT data from the first slot to the output. (and damage if the first slot and output are damageable items) You can remove recipes using the "remove" function and an ItemStack.
    # Currently CraftTweaker support is limited, so some features from CraftTweaker may not work. .noReturn() and .anyDamage().onlyDamaged() are known to not work.
    # However anyDamage() by itself does work, and is required for tool inputs to actually copy their damage over to the output.
    # Examples:
    # mods.etfuturum.smithingTable.addRecipe(<IC2:itemToolBronzeSword>, <minecraft:iron_sword>.anyDamage(), <ore:ingotBronze>); //(Use "ingotBronze" on iron sword to convert it to a bronze sword)
    # mods.etfuturum.smithingTable.addRecipeNoNBT(<etfuturum:sponge:1>, <minecraft:sponge>, <minecraft:water_bucket>); //Take a vanilla sponge and a water bucket, you will get an Et Futurum Requiem wet sponge. [default: true]
    B:enableSmithingTable=true

    # Will attempt to seek and auto-add recipes to itself. Look at ConfigFunctions.cfg "autoAddSmoker" for more info.
    # Compatible with CraftTweaker. In the same way that you'd use furnace.addRecipe or furnace.remove, you can use "mods.etfuturum.smoker" instead of "furnace". [default: true]
    B:enableSmoker=true

    #  [default: true]
    B:enableSponge=true

    # Currently DOES NOT HAVE ANY FUNCTIONALITY. Decoration ONLY! [default: true]
    B:enableStonecutter=true

    # Enables target block from 1.16 [default: true]
    B:enableTarget=true

    # Enables wood variant trapdoors [default: true]
    B:enableTrapdoors=true

    # Enables wood variant buttons and pressure plates [default: true]
    B:enableWoodRedstone=true
}


##########################################################################################################
# misc blocks
#--------------------------------------------------------------------------------------------------------#
# Blocks that don't fit in any other category.
##########################################################################################################

"misc blocks" {
    #  [default: true]
    B:enableBanners=true

    # Enables log blocks with bark on all sides [default: true]
    B:enableBarkLogs=true

    # A solid, indestructible and invisible block. Can be seen when holding it in Creative mode. [default: true]
    B:enableBarrier=true

    # This must be on for the basalt deltas biome to generate unless Netherlicious is installed. [default: true]
    B:enableBlackstone=true

    #  [default: true]
    B:enableChain=true

    #  [default: true]
    B:enableConcrete=true

    # Copper sub-blocks and items. Disable copper but keep this on if you want the new copper items and blocks made of it, without the main ingot, ore or copper block itself. [default: true]
    B:enableCopperSubItems=true

    # Slabs for vanilla blocks: stone, mossy stone brick, mossy cobble, cut sandstone [default: true]
    B:enableExtraVanillaSlabs=true

    # Stairs for vanilla blocks: stone, mossy stone brick, mossy cobble [default: true]
    B:enableExtraVanillaStairs=true

    # Stairs for vanilla blocks: stone brick, mossy stone brick, sandstone, brick, nether brick [default: true]
    B:enableExtraVanillaWalls=true

    # Enables wood variant fences and gates [default: true]
    B:enableFences=true

    #  [default: true]
    B:enableGlazedTerracotta=true

    # Enables honey blocks, honeycomb blocks, honeycombs, and honey bottles. See entities.cfg for toggling bee nests, beehives, and bees. [default: true]
    B:enableHoney=true

    #  [default: true]
    B:enableLantern=true

    # Invisible light blocks. Only has a selection box when held, right click to change light level. Otherwise functionally identical to air and can be replaced by placing blocks into it. Invisible, but can be seen when holding it in Creative mode. [default: true]
    B:enableLightBlock=true

    #  [default: true]
    B:enableQuartzBricks=true

    # Note: Also enables cracked and chiseled nether bricks as they use this ID too [default: true]
    B:enableRedNetherBricks=true

    #  [default: true]
    B:enableSmoothQuartz=true

    #  [default: true]
    B:enableSmoothSandStone=true

    #  [default: true]
    B:enableSmoothStone=true

    # Enables stripped log blocks [default: true]
    B:enableStrippedLogs=true
}


##########################################################################################################
# misc items
#--------------------------------------------------------------------------------------------------------#
# Items that don't fit in any other category.
##########################################################################################################

"misc items" {
    #  [default: true]
    B:enableBeetroot=true

    #  [default: true]
    B:enableIronNugget=true

    #  [default: true]
    B:enableLingeringPotions=true

    #  [default: true]
    B:enableMutton=true

    #  [default: true]
    B:enableNewDyes=true

    # Appears in stronghold corridor and dungeon chests. [default: true]
    B:enableOtherside=true

    # Appears in Nether fortress chest loot. [default: true]
    B:enablePigstep=true

    # If true, vanilla and Et Futurum copper ores will drop raw ore items. [default: true]
    B:enableRawOres=true

    #  [default: true]
    B:enableSuspiciousStew=true

    #  [default: true]
    B:enableTippedArrows=true

    #  [default: true]
    B:enableTotemUndying=true
}


##########################################################################################################
# natural blocks
#--------------------------------------------------------------------------------------------------------#
# Blocks that can generate naturally in your world. Check world.cfg for generation values.
##########################################################################################################

"natural blocks" {
    # Enables tinted glass, amethyst blocks, budding amethyst and amethyst crystals. Also enables the item too. [default: true]
    B:enableAmethyst=true

    # Enables bamboo wood and all of its wood subtypes, including the bamboo stalks themselves. [default: true]
    B:enableBambooBlocks=true

    # This must be on for the basalt deltas biome to generate unless Netherlicious is installed. [default: true]
    B:enableBasalt=true

    #  [default: true]
    B:enableBlueIce=true

    #  [default: true]
    B:enableBoneBlock=true

    #  [default: true]
    B:enableCalcite=true

    # Enables cherry wood and all of its wood subtypes. [default: true]
    B:enableCherryBlocks=true

    # Enables chorus plants and purpur blocks [default: true]
    B:enableChorusBlocks=true

    #  [default: true]
    B:enableCoarseDirt=true

    # Copper ore and copper blocks, variants, and waxed variants. (Slime balls are used if no mod introduces wax and if honey is disabled) [default: true]
    B:enableCopper=true

    #  [default: true]
    B:enableCornflower=true

    #  [default: true]
    B:enableCryingObsidian=true

    #  [default: true]
    B:enableDeepslate=true

    # Enable deepslate ores for copper ore and vanilla ores when deepslate generates over them. [default: true]
    B:enableDeepslateOres=true

    #  [default: true]
    B:enableGrassPath=true

    #  [default: true]
    B:enableLilyOfTheValley=true

    #  [default: true]
    B:enableMagmaBlock=true

    # Enables mud, packed mud and mud bricks, as well as the mud brick stairs, slabs and walls. [default: true]
    B:enableMud=true

    #  [default: true]
    B:enableNetherGold=true

    #  [default: true]
    B:enableNetherite=true

    #  [default: true]
    B:enableNetherwartBlock=true

    #  [default: true]
    B:enablePrismarine=true

    #  [default: true]
    B:enableRedSandstone=true

    # Soul torches and soul lanterns. [default: true]
    B:enableSoulLighting=true

    # Not required for the Soul Sand Valley to generate. [default: true]
    B:enableSoulSoil=true

    # Enable Granite/Andesite/Diorite [default: true]
    B:enableStones=true

    #  [default: true]
    B:enableSweetBerryBushes=true

    #  [default: true]
    B:enableTuff=true

    #  [default: true]
    B:enableWitherRose=true

    # Set this to false to easily turn off all Nether blocks. This also turns off all Nether biomes because they require the blocks to generate.
    # My biomes have compat with Netherlicious (read world.cfg for more info) but if you don't want any compat at all turn this off.
    # This disables the following toggles: enableCrimsonBlocks, enableWarpedBlocks, enableBlackstone, enableSoulSoil, enableSoulLighting and enableBasalt.
    # Amethyst geodes use smooth basalt so go to world.cfg to change the outer block to something else or they won't generate.
    # This also turns off Nether wart blocks even though they are older, because Netherlicious also has those. [default: true]
    B:masterNetherToggle=true
}


```

**Comment by mitchej123** [comment]
Tl:DR - It backports stuff from newer versions of MC

![image](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/assets/1894689/cc54b4be-70c4-46c2-a9b8-8baab39a1457)
![image](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/assets/1894689/61ab125f-7b0e-41d2-85e0-5013641ef695)
![image](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/assets/1894689/7181f3cd-fddf-404e-88d6-a195d36a2660)

**Comment by mitchej123** [comment]
I'd probably vote for disabling the new bees, I don't really like them.  As far as elytra  recipe/drop - sure.  @roadhog360 is open to PRs, and configs for GTNH, and I'm getting him setup on the NH maven under thirdparty so we can depend on his mod and modify recipes in coremod

**Comment by chochem** [comment]
thanks!
so yea. elytra, netherite, shulkers might be the big ones to think about and/or ask dream about as to how we want to place or use them in the pack.
But also a bunch of other smaller things like lets say a deepslate gt dust that can be centrifuged for some stuff and things like that.

**Comment by NotAPenguin0** [comment]
I am personally not a fan of deepslate in worldgen at all, it takes absolutely forever to dig through even with good tools. That said, you can bypass that issue mostly with tinkers tools and GT miners, so maybe this isn't really a problem in GTNH.

**Comment by mitchej123** [comment]
> I am personally not a fan of deepslate in worldgen at all, it takes absolutely forever to dig through even with good tools. That said, you can bypass that issue mostly with tinkers tools and GT miners, so maybe this isn't really a problem in GTNH.

Could always adjust the hardness of it, it'd be more for aesthetics if anything.  Also curious how it looks compared to black granite/basalt, etc.

**Comment by PlayfulPiano** [comment]
In regards to Elytra, it might be worthwhile for it to follow the path of like, mantle of the raven / hang glider. Actually, the mantle being a crafting component could make sense.

**Comment by Roadhog360** [comment]
Yeah new bees AI is kinda crummy and isn't really meant to be used with other advanced bee mods anyways

**Comment by jss2a98aj** [comment]
The EFR Elytra has optional integration with Baubles Expanded if it taking up the chest slot is found to be too much of a downside when balancing and integrating things.

**Comment by Caedis** [comment]
Per Modern Controlling, ModernKeyBinding, KeyboardWizard-Legacy

Should they all be merged into our Controlling fork or?

**Comment by mitchej123** [comment]
> Per Modern Controlling, ModernKeyBinding, KeyboardWizard-Legacy
> 
> Should they all be merged into our Controlling fork or?

I'm not opposed If someone wants to take the lead on it

**Comment by PlayfulPiano** [comment]
Possibility of adding CraftingTweaks to that list? #15305 
It's MIT for the 1.7.10 version, had no issues running it personally. Ability to rotate/spread/balance your crafting grid would be pretty nice imo.

**Comment by Dream-Master** [comment]
> Possibility of adding CraftingTweaks to that list? #15305 It's MIT for the 1.7.10 version, had no issues running it personally. Ability to rotate/spread/balance your crafting grid would be pretty nice imo.

We need to contact the mod developer to be sure that older MC versions are under MIT license before we can use them.

**Comment by Dream-Master** [comment]
Ar Village Names still compatible with witchery generated villages? That villages need a facelift is a nice idea.

**Comment by PlayfulPiano** [comment]
> > Possibility of adding CraftingTweaks to that list? #15305 It's MIT for the 1.7.10 version, had no issues running it personally. Ability to rotate/spread/balance your crafting grid would be pretty nice imo.
> 
> We need to contact the mod developer to be sure that older MC versions are under MIT license before we can use them.

Got it, will ask to clarify about the license.

**Comment by PlayfulPiano** [comment]
Update on craftingtweaks:
![image](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/assets/3158887/e9ea8812-c43a-4c13-93c1-11331821eb74)

**Comment by Caedis** [comment]
So if we fork, it gets a new name. `Tweaks for Crafting` fits the requirements :trollface:

**Comment by ah-OOG-ah** [comment]
did anyone have to change gadomancy config as well? gadomancy's PotionMiningLuck keeps getting a conflicting ID

**Comment by ah-OOG-ah** [comment]
Also, Alfheim slimes seem to spawn like crazy, so I removed it and ASJCore.
Also also, I noticed that items seem to have slow fall (with or without Alfheim).

**Comment by Roadhog360** [comment]
> Also, Alfheim slimes seem to spawn like crazy, so I removed it and ASJCore.
> Also also, I noticed that items seem to have slow fall (with or without Alfheim).

May be related to [Et-Futurum-Requiem/#488](https://github.com/Roadhog360/Et-Futurum-Requiem/issues/448)

**Comment by brandyyn** [comment]
> Also, Alfheim slimes seem to spawn like crazy, so I removed it and ASJCore. Also also, I noticed that items seem to have slow fall (with or without Alfheim).

Just disable the elemental slimes, they are not required for anything. I personally disabled them

**Comment by PlayfulPiano** [comment]
https://www.curseforge.com/minecraft/mc-mods/stackie
Consideration for this too? Basically a precursor to "clumps" in modern versions. Should help with xp spam lag, as well as being an MIT license.

**Comment by huihiuhuai** [comment]
Malisis Doors semm to have issues with EFR and VN. The animated doors generate weirdly in village
![图片](https://github.com/user-attachments/assets/2db6a1ec-b1f1-463c-9d66-21d11ba0d4a4)
![19a3d0d59c547c7621fc0ff72e25b2c](https://github.com/user-attachments/assets/055591f9-1cf8-4c55-9499-01be29d6f693)
 and EFR's new door variants are not animated of course.

**Comment by huihiuhuai** [comment]
Malisis Doors semm to have issues with EFR and VN. The animated doors generate weirdly in village
![图片](https://github.com/user-attachments/assets/f7b982c0-8d7a-45f6-bb0a-19c57e7aa4c9)
![19a3d0d59c547c7621fc0ff72e25b2c](https://github.com/user-attachments/assets/2c489ad7-4096-4e98-b840-0ca80d2391ed)
and EFR's new door variants are not animated of course.

**Comment by Wmaxlees** [comment]
Alternate wood doors are already supported via . We can disable doors in Malisis' Doors.

We can disable doors in EFR in the `blocksitems.cfg` file.

B:enableDoors=false

**Comment by huihiuhuai** [comment]
> Alternate wood doors are already supported via Malisis' Doors. Same with trap doors.

lol you forgot cherry and mangrove doors

**Comment by Dream-Master** [comment]
ah it remembers me to test this stuff an configure it for GTNH

**Comment by Wmaxlees** [comment]
Okay. Here's actually a more comprehensive list of TODOs for EFR AFAICT.

### General Issues
#### Raw ores
EFR makes vanilla ores drop ore chunks. This shouldn't be an issue because raw ores don't spawn but we can also just disable this feature entirely.

```
// blocksitems.cfg
B:enableRawOres=false
```

#### Bees
GTNH already has extensive bee support for non-vanilla bees. I'd suggest we just disable this.

```
// blocksitems.cfg
B:enableHoney=false
```

```
// entities.cfg
B:enableBees=false
```

#### Deepslate Ores
Deepslate now spawns naturally in large sections underground. This does no overwrite ore veins but the ores look out of place as normal stone in a big blob of deepslate.

Do we need to create gregtech deepslate ore?

### Already Supported
#### Amethyst
Amethyst is already a material in the game. It can be disabled in EFR.

```
// blocksitems.cfg
B:enableAmethyst=false
```

#### Doors, Gates, Trapdoors
Modern versions of doors, trapdoors, and fence gates are all already provided by Malisis' Doors and are gregified. We can disable doors and trapdoors in the EFR config.

```
// blocksitems.cfg 
B:enableDoors=false
B:enableTrapdoors=false
```

Fences and fence gates are tied to the same config in EFR so they'll need to be split in the config so we can just disable the gates and keep the fences.

#### Stones
Diorite, andesite, and granite already supported by Botania + Chisel. We can turn off their spawning in EFR.

```
// blocksitems.cfg
B:enableStones=false
```

### Recipes
This is a list of items provided by EFR that will need their recipes to match the existing versions.

- Wooden buttons
- Colored Beds
- Wooden Fences
- Pressure Plates
- Slabs
- Concrete/Concrete Powder
- Boats
- Signs

There are also items that should probably just be harder to craft or don't currently have a recipe.

- *Block of Copper*: Probably should be a compressor recipe
- *Smooth Stone*: Doesn't have a recipe
- *Slime Block*: Very similar to Congealed Slime. Compressor maybe?
- *Blue ice*: Match Packed Ice and have a compressor recipe
- *Barrel*: Should have similar crafting difficult to a chest
- *Boats with a chest*: Maybe more complex than just a boat and a chest?

### Pollution
A couple of blocks need to produce pollution while running to match the furnace.

- Blast Furnace
- Smoker

### Progress Concerns
There are a handful of items that have a mechanical impact on the game. I'm not familiar enough with GTNH to know if these will break progression so any feedback here is welcome. All of these items can be configured to be off in the EFR mod.

#### Composter
Crafting: Some wooden slabs

Mechanic: Feed the composter plants and it produces Bonemeal

#### Wither Rose
Crafting: Get a wither to kill a passive mob and the mob drops a Wither Rose

Mechanic: When an entity walks through the Wither Rose, they receive the Wither effect

#### Shulker Box
Crafting: Get Shulker Shells from a mob that spawns in End Cities. Craft them together with a chest.

Mechanic: A chest that keeps its contents when broken.

#### Tinted Glass
Crafting: 4 Amethyst + 1 Glass

Mechanic: Glass that doesn't allow light to pass through. Useful to see into mob farms + dark rooms.

#### Observer
Crafting: Redstone + Nether Quartz + Cobblestone

Mechanic: Block produces a 1 tick redstone signal any time a block is updated in front of it. Has tons of practical redstone uses.

#### Target Block
Crafting: 4 Redstone + Hay Bail

Mechanic: Produces a redstone signal when hit with a projectile. Also conducts redstone signal through itself. Because of that last fact, it has a ton of practical redstone uses.

#### Elytra
Crafting: Found in the end.

Mechanic: Allows gliding and, when paired with rockets, provides flight.

#### Boat with a Chest
Crafting: Boat + Chest in crafting table.

Mechanic: A movable chest. Makes exploring + looting easier.

#### Netherite gear
Crafting: Diamond tier gear + a Netherite Ingot

Mechanic: Gear that is better than diamond.

#### Lingering potions
Crafting: Any splash potion + Dragons Breath (from the End Dragon)

Mechanic: The potion now leaves an area of effect.

#### Tipped Arrows
Crafting: Arrows + Lingering Potion

Mechanic: The arrow applies the potion affect on hit.

#### Totem of Undying
Crafting: Looted from Pillagers

Mechanic: If holding the Totem, if the player would die, they instead destroy the totem and get a bunch of health

#### Nether Gold
Nether gold spawns naturally in Netherrack that drops gold nuggets.

**Comment by NotAPenguin0** [comment]
> Tinted Glass
> Crafting: 4 Amethyst + 1 Glass
> Mechanic: Glass that doesn't allow light to pass through. Useful to see into mob farms + dark rooms.

Well, I didn't know this was a thing in modern minecraft, but aside from that we do already have a Tinted Glass block in ZTones. I'm not sure if it provides the same functionality though.

**Comment by Caedis** [comment]
Can probably disable Tinted Glass since I believe we have dark glass and dark ethereal glass from...Extra Utils?

**Comment by EnderProyects** [comment]
> #### Bees
> GTNH already has extensive bee support for non-vanilla bees. I'd suggest we just disable this.

I think we can keep this enable, since they can be used in new bee stuf mechanics ( as well as an interesting way of produce small amounts of honney)

**Comment by Dynamiczbee** [comment]
Additional suggestions with EFR based on my playtesting in Nightly 594 w/ the addition of @Wmaxlees configs. 
Modern copper ore still spawns randomly in stone, should either be added to a different vein or make a crafting recipe for it?
Sheep drop mutton from Pam's and EFR, could be left as is or merge them.
There is a number of questions regarding how EFR Bamboo generates along RWG, note the conversation in the discord channel starting [here](https://discord.com/channels/181078474394566657/603348502637969419/1274491572976881717). Looks like, and this is a pretty uneducated read so please check Chochem and others' messages, there is a biome ID mixup with generation, but even if that is not the case the generation currently does not look good, at all, so I would still suggest a change here personally.
The blast furnace and smoker recipes while IMO fine, should probably be gregified. Especially the blast furnace.
I'll add any other comments I get from playing through here but let me know if I should open a seperate issue, and if that is the case under which repo.

**Comment by CatsLeftEar** [comment]
Suggest adding a mod Axiom. An overpowered worldedit with many tools

**Comment by Caedis** [comment]
> Suggest adding (backporting) a mod Axiom. An overpowered worldedit with many tools. Usage is limited for singleplayer only basically, for easier building.

We already have something in progress for a better worldedit

**Comment by tangzixiang-hash** [comment]
I need the toolbox from create, I find it very useful, especially when I set its working range to be very large, and then input components such as AE2 cables and buses into it, I can easily lay out the network, and its blueprint, just click to synthesize that thing, is also super easy to use

**Comment by PlayfulPiano** [comment]
> I just want +1 Village Names, specifically for its ability to prevent villages within X chunks of spawn. I hate landing in a new world only to cross one hill, and surprise! its a village. 

... And that's a negative because?

**Comment by darchitectlp** [comment]
Eh, I like making stuff hard. Walking over said hill and having access to a thaumcraft villager to buy thaumium from, tear down their smeltery and railcraft station, steal some harvestcraft crops... I just don't enjoy that. I wanna hafta be far enough along that I can feel I can safely travel a thousand blocks in the hopes of maybe finding a village. It's a risk/reward thing.

I like being force to explore. Regardless, I feel like villagers always kinda break the game, modded or not. But really, the mod adds so many other great things that this would be just a small thing I'd like.

**Comment by Ladaj22** [comment]
https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/21303

**Comment by PlayfulPiano** [comment]
I'd take it that the keybinding mods will be pushed to 2.9 for implementation purposes? Or is there a possibility for them to be added in a minor version of 2.8?

**Comment by thenextelement** [comment]
I have been playing 2.8, and especially with backhand and EFR added I think it would make a lot of sense to add both [Aqua Acrobatics Legacy](https://github.com/RagexPrince683/aquaacrobatics) for 1.13+ swimming, as well as [The Ears mod](https://modrinth.com/mod/ears/versions)  which backports modern 3D skin layers.
I have not tested Ears in a GTNH Instance, but it is lightweight and works great in a 1.7.10 Magic Pack I have been working on for fun, also added AAL to my 2.8 Instance and can confirm it works.
Edit: Ears mod as of now has a bug in which the Guardian of Gaia and all player head blocks/items do not render properly, unsure if fixed but will report if possible to ears devs.

**Comment by Omgise** [comment]
I think adding [Antiquities](https://github.com/PufferTeam-ModArchive/Antiquities/) would also be great. It has better performance than BiblioCraft and a lot of potential

**Comment by User198263321** [comment]
A mod or way to increase the sound limit? I tried dynamic surroundings but it does not seem to work.

**Comment by Caedis** [comment]
> I think adding [Antiquities](https://github.com/PufferTeam-ModArchive/Antiquities/?rgh-link-date=2025-10-06T11%3A11%3A16.000Z) would also be great. It has better performance than BiblioCraft and a lot of potential

We are already aware of that mod and its very likely

**Comment by choasgod2003** [comment]
could we add the <a href="https://github.com/TwelveIterationMods/Waystones/tree/1.7.10">waystones</a> mod and make the waystones craftable around mid-late game.
the <a href="https://github.com/CompactMods/CompactMachines/tree/1.7.10">compact machines</a> mod for the field shrinkers for some interesting recipes.
<a href="https://github.com/GTNewHorizons/The-Aether-GTNH">the aether</a> mod because the aether is really cool but instead of the portal could we try to add it to the tier 1 rocket also it is already for GTNH.

**Comment by User198263321** [comment]
Support for modded mob heads in Journeymap?

**Comment by yhdsl** [comment]
The [Salutation](https://github.com/Speiger/Salutation) mod (requires [Carbon Config](https://github.com/Carbon-Config-Project/CarbonConfig) as a prerequisite) is a backward port of the command prompt system from 1.19.2+.

![CFImage](https://raw.githubusercontent.com/Speiger/Salutation/refs/heads/1.7.10/CFImage.png)

As shown in the image, it significantly improves command input.

**Comment by Xiarno** [comment]
Backport of modern SFM where it's coding based instead of visual code base.
